app/serializers/article.js

```diff
@@ -17,6 +17,9 @@ export default DS.RESTSerializer.extend(DS.EmbeddedRecordsMixin, {
       });
       return newArticle;
     });
+    newPayload.meta = {
+      articlesCount: payload.articlesCount
+    };
```

Comment:
This seems to cause a warning:
Encountered "articlesCount" in payload, but no model was found for model name "articles-count" (resolved model name using (unknown).modelNameFromPayloadKey("articlesCount"))

Can you look into this please?